sinks/external.go

```diff
@@ -26,6 +27,7 @@ import (
 type externalSinkManager struct {
 	decoder       sink_api.Decoder
 	externalSinks []sink_api.ExternalSink
+	cache         cache.Cache
```

Comment:
I was thinking of keeping the cache as part of heapster manager instead of here since it holds data that is not in metrics form.